Assets/Labrys/Editor/FeatureEditor/FeatureEditorWindow.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Labrys.Editor.FeatureEditor.Panels;
+using Labrys.Editor.FeatureEditor.Tools;
+using Labrys.FeatureEditor;
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEngine;
+
+namespace Labrys.Editor.FeatureEditor
+{
+	public class FeatureEditorWindow : EditorWindow
+	{
+		private const string EDITOR_PREF_SETTINGS = "FeatureEditorWindowSettings";
+
+		private static FeatureEditorWindow instance;
+
+		public static FeatureEditorWindow GetInstance()
+		{
+			if(instance == null)
+			{
+				instance = OpenWindow();
+			}
+			return instance;
+		}
+
+		[MenuItem("Window/Labrys Feature Editor")]
+		private static FeatureEditorWindow OpenWindow()
+		{
+			FeatureEditorWindow window = GetWindow<FeatureEditorWindow> ();
+			window.titleContent = new GUIContent ("Labrys Feature Editor");
+			return window;
+		}
+
+		[OnOpenAsset(1)]
+		private static bool OnOpenAsset(int instanceId, int line)
```

Comment:
unused parameters

Reply:
They're part of the contract to use the attribute `OnOpenAsset`.  I don't need them, but they have to be there.

Reply:
Figured as much. I'll tell the IDE to shush.